llm/server.go

```diff
@@ -33,6 +34,33 @@ import (
 	"github.com/ollama/ollama/model"
 )
 
+type filteredEnv []string
+
+func (e filteredEnv) LogValue() slog.Value {
```

Comment:
`LogValue` is nice since it will only be called if the log will be outputted. It's similar to the condition previously which checks if it should output before filtering